File: src/main/java/org/customer/dto/AccountDTO.java
package org.customer.dto;

import lombok.Builder;
import lombok.Data;
import java.util.ArrayList;
import java.util.List;

@Builder
@Data
public class AccountDTO {
    private String id;
    private String customerId;
    private double balance;
    private List<TransactionDTO> transactions = new ArrayList<>();
}

File: src/main/java.kotlin/org/customer/dto/AccountDTO.kt
package org.customer.dto

import lombok.Builder
import lombok.Data

@Builder
@Data
class AccountDTO {
    private val id: String? = null
    private val customerId: String? = null
    private val balance = 0.0
    private val transactions: MutableList<TransactionDTO?> = ArrayList()
}
